src/test/java/training/article/update.feature

```diff
@@ -0,0 +1,34 @@
+Feature: update an article
+
+  Background:
+    * url url
+    * def login = call read('../sign_in/sign_in.feature@loginSuccessfully')
+    * def username = username
+    * def token = "Bearer " + login.response.user.token
+    * def getArticles = call read('getAll.feature@getArticles')
+    * def randomArticle = function(articles) { return articles[Math.floor(Math.random() * articles.length)] }
```

Comment:
you are repeating this function in some features. Please put this logic on a common file.

Reply:
you can create a package `helpers` and put in them this function.